deps.qt/qt6.ps1

```diff
@@ -1,132 +1,281 @@
-        $QtBuildConfiguration = '-release -force-debug-info'
-    } elseif ( $Configuration -eq 'Debug' ) {
-        $QtBuildConfiguration = '-debug'
-    } elseif ( $Configuration -eq 'MinSizeRel' ) {
-        $QtBuildConfiguration = '-release'
+    $TargetCPUs = @{
+        x64 = 'x86_64'
+    }
+
+    $OnOff = @('OFF', 'ON')
+    $Options = @(
+        $CmakeOptions
+        "-DBUILD_SHARED_LIBS:BOOL=$($OnOff[$script:Shared.isPresent])"
+    )
+
+    if ( $Config -eq 'RelWithDebInfo' ) {
+        $Options += @(
+            '-DFEATURE_separate_debug_info:BOOL=ON'
+        )
     }
 
-    $BuildCommand = "..\..\..\qt6\configure -opensource -confirm-license ${QtBuildConfiguration} -nomake examples -nomake tests -schannel -no-dbus -no-freetype -no-icu -no-openssl -no-feature-androiddeployqt -no-feature-pdf -no-feature-printsupport -no-feature-qmake -no-feature-sql -no-feature-testlib -no-feature-windeployqt -qt-doubleconversion -qt-pcre -qt-zlib -qt-libjpeg -qt-libpng -qt-libmd4c -qt-tiff -qt-webp -DQT_NO_PDF -DCMAKE_IGNORE_PREFIX_PATH=C:/Strawberry/c -prefix ${BuildPath}"
+    $Options += @(
+        '-DFEATURE_androiddeployqt:BOOL=OFF'
+        '-DFEATURE_brotli:BOOL=OFF'
+        '-DFEATURE_cups:BOOL=OFF'
+        '-DFEATURE_dbus:BOOL=OFF'
+        '-DFEATURE_doubleconversion:BOOL=ON'
+        '-DFEATURE_freetype:BOOL=OFF'
+        '-DFEATURE_harfbuzz:BOOL=OFF'
+        '-DFEATURE_glib:BOOL=OFF'
+        '-DFEATURE_icu:BOOL=OFF'
+        '-DFEATURE_itemmodeltester:BOOL=OFF'
+        '-DFEATURE_libjpeg:BOOL=ON'
+        '-DFEATURE_libpng:BOOL=ON'
+        '-DFEATURE_macdeployqt:BOOL=OFF'
+        '-DFEATURE_openssl:BOOL=OFF'
+        '-DFEATURE_pcre2:BOOL=ON'
+        '-DFEATURE_pdf:BOOL=OFF'
+        '-DFEATURE_printsupport:BOOL=OFF'
+        '-DFEATURE_printer:BOOL=OFF'
+        '-DFEATURE_printdialog:BOOL=OFF'
+        '-DFEATURE_printpreviewdialog:BOOL=OFF'
+        '-DFEATURE_printpreviewwidget:BOOL=OFF'
+        '-DFEATURE_qmake:BOOL=OFF'
+        '-DFEATURE_schannel:BOOL=ON'
+        '-DFEATURE_sql:BOOL=OFF'
+        '-DFEATURE_system_doubleconversion:BOOL=OFF'
+        '-DFEATURE_system_libjpeg:BOOL=OFF'
+        '-DFEATURE_system_libpng:BOOL=OFF'
+        '-DFEATURE_system_pcre2:BOOL=OFF'
+        '-DFEATURE_system_zlib:BOOL=OFF'
+        '-DFEATURE_testlib:BOOL=OFF'
+        '-DFEATURE_textmarkdownreader:BOOL=OFF'
+        '-DFEATURE_textmarkdownwriter:BOOL=OFF'
+        '-DFEATURE_windeployqt:BOOL=OFF'
+        '-DQT_BUILD_BENCHMARKS:BOOL=OFF'
+        '-DQT_BUILD_EXAMPLES:BOOL=OFF'
+        '-DQT_BUILD_EXAMPLES_BY_DEFAULT:BOOL=OFF'
+        '-DQT_BUILD_MANUAL_TESTS:BOOL=OFF'
+        '-DQT_BUILD_TESTS:BOOL=OFF'
+        '-DQT_BUILD_TESTS_BY_DEFAULT:BOOL=OFF'
+        '-DQT_BUILD_TOOLS_BY_DEFAULT:BOOL=OFF'
+        '-DCMAKE_IGNORE_PREFIX_PATH:PATH=C:/Strawberry/c'
+    )
+
+    $Options = ($Options -join ' ') -replace '-G Visual Studio \d+ \d+','-G Ninja' -replace "-A ${Target}",''
+
+    Log-Information "Config qtbase (${Target})"
```

Comment:
The text previously said "Configure" which appears consistent with other build scripts (see `deps.windows/20-freetype.ps1`, for example). I don't know why qt6.zsh says "Config" other than perhaps an oversight.